<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelationMng</class>
 <widget class="QDialog" name="RelationMng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户关系管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>数据查询</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,3,3,1,1">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>关系类别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cm_rela_type"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
        <item>
         <widget class="QPushButton" name="btnRela">
          <property name="text">
           <string>选择关系人</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="la_rela">
          <property name="text">
           <string>未选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
        <item>
         <widget class="QPushButton" name="btnClient">
          <property name="text">
           <string>选择客户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="la_client">
          <property name="text">
           <string>未选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空参数</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>必须选择一个关系类别才可以查询，若要编辑关系，必须选择关系人以及客户，执行查询操作后，才可对列表进行增删改查操作</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvIns">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,5,1,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnGoFirst">
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGoPre">
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGoNext">
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGoLast">
         <property name="text">
          <string>尾页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="l_count">
       <property name="text">
        <string>共0页</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,5,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnGoNum">
         <property name="text">
          <string>跳转至</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lPageNum"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
